runtime/queries/bass/highlights.scm

```diff
@@ -0,0 +1,98 @@
+; GENERATED VIA https://github.com/vito/tree-sitter-bass
+
+(comment) @comment.line
+
+["(" ")" "[" "]" "{" "}"] @punctuation.bracket
+
+[(ignore) (null) (bool)] @constant.builtin
+
+(int) @constant.numeric
+
+(string (string_escape) @constant.character.escape)
+
+(string) @string
+
+[(command) (path) (relpath)] @namespace
+
+(keyword) @string.special.symbol
+
+
+((list . (symbol) @keyword.control.conditional (#match? @keyword.control.conditional "^(if|case|cond|when)$"))
+ (set! "priority" 105))
```

Comment:
Is the priority predicate something from nvim-treesitter? I don't think we have it implemented and I don't think it should be necessary precedence-wise

Reply:
Yeah, this is just needed for Neovim. It doesn't seem to cause any harm but I can adjust the template generation to only include it for Neovim, since it's definitely a bit odd to have it here (folks might look to this as an example and get the wrong idea).

Thanks for allowing this to be merged (once I tidy it up). :slightly_smiling_face: